appveyor.yml

```diff
@@ -0,0 +1,34 @@
+version: 1.0.{build}-{branch}
+
+environment:
+  matrix:
+    - RUBY_VERSION: 25
+    - RUBY_VERSION: 25
+      CAPYBARA_FF: true
```

Comment:
Really don't need to build on all the ruby versions on Windows (if it's a ruby version specific issue it'll fail on the linux build too)-- just go for minimum (2.2.2)  and latest - also don't really need to do FF on every ruby version, just pick one of them.